### PR TITLE
BXMSDOC-6564-7.39.x: Add note regarding DMN support

### PR DESCRIPTION
**Pull request by csherrar**

### CODE REVIEW

File: doc-content/enterprise-only/installation/patches-applying-proc.adoc [hunk 1]
```diff
@@ -6,6 +6,11 @@ Automated update tools are often provided with both patch updates and new minor
 
 You can use the same automated update tool to apply both patch updates and minor release upgrades to {PRODUCT} {PRODUCT_VERSION}. Patch updates of {PRODUCT}, such as an update from version {PRODUCT_VERSION} to {PRODUCT_VERSION}.1, include the latest security updates and bug fixes. Minor release upgrades of {PRODUCT}, such as an upgrade from version 7.7.x to 7.8, include enhancements, security updates, and bug fixes.
 
+[IMPORTANT]
+====
+{PRODUCT} 7.9 provides design and runtime support for DMN 1.2 models at conformance level 3, and runtime-only support for DMN 1.1 and 1.3 models at conformance level 3. Any DMN 1.1 and 1.3 models that you import into {CENTRAL}, open in the DMN designer, and save are converted to DMN 1.2 models. DMN 1.1 and 1.3 models are not supported in the DMN designer in {CENTRAL}.
```
Comment (karreiro) [inline doc-content/enterprise-only/installation/patches-applying-proc.adoc:11]: Actually, we're not currently converting models into DMN 1.2. Thus, I suggest this sentence (reference: https://github.com/kiegroup/kie-docs/pull/2827):

```suggestion
{PRODUCT} 7.9 provides design and runtime support for DMN 1.2 models at conformance level 3, and runtime-only support for DMN 1.1 and 1.3 models at conformance level 3. You can design your DMN models directly in {CENTRAL} or import existing DMN models into your {PRODUCT} projects for deployment and execution. DMN 1.1 and 1.3 models are currently not supported in the DMN designer in {CENTRAL}.
```

**Review by karreiro — COMMENTED** [review]
Thank you, @csherrar. I've left an inline comment :-)

File: doc-content/enterprise-only/installation/patches-applying-proc.adoc (hunk 1 above)
Comment (csherrar) [reply doc-content/enterprise-only/installation/patches-applying-proc.adoc:11]: @karreiro Thanks. This is based on an existing statement. I'll copy it below. Should this be corrected?

{PRODUCT} provides design and runtime support for DMN 1.2 models at conformance level 3, and includes enhancements and fixes to FEEL and DMN model components to optimize the experience of implementing DMN decision services with {PRODUCT}. {PRODUCT} also provides runtime-only support for DMN 1.1 and 1.3 models at conformance level 3, but any DMN 1.1 models that you import into {CENTRAL}, open in the DMN designer, and save are converted to DMN 1.2 models. DMN 1.3 models are not supported in the DMN designer in {CENTRAL}.

Comment (karreiro) [reply doc-content/enterprise-only/installation/patches-applying-proc.adoc:11]: @csherrar actually, we need to change this part "**you import into {CENTRAL}, open in the DMN designer, and save are converted to DMN 1.2 models**". We're not converting 1.1 models anymore. This is the temporary limitation we're warning users :)

Comment (csherrar) [reply doc-content/enterprise-only/installation/patches-applying-proc.adoc:11]: Got it, thanks! Will make those changes now

File: doc-content/enterprise-only/installation/patches-applying-proc.adoc [hunk 2]
```diff
@@ -6,6 +6,11 @@ Automated update tools are often provided with both patch updates and new minor
 
 You can use the same automated update tool to apply both patch updates and minor release upgrades to {PRODUCT} {PRODUCT_VERSION}. Patch updates of {PRODUCT}, such as an update from version {PRODUCT_VERSION} to {PRODUCT_VERSION}.1, include the latest security updates and bug fixes. Minor release upgrades of {PRODUCT}, such as an upgrade from version 7.7.x to 7.8, include enhancements, security updates, and bug fixes.
 
+[IMPORTANT]
+====
+{PRODUCT} 7.9 provides design and runtime support for DMN 1.2 models at conformance level 3, and runtime-only support for DMN 1.1 and 1.3 models at conformance level 3. You can design your DMN models directly in {CENTRAL} or import existing DMN models into your {PRODUCT} projects for deployment and execution. DMN 1.1 and DMN 1.3 models are not currently supported in the DMN designer in {CENTRAL}.
```
Comment (michelehaglund) [inline doc-content/enterprise-only/installation/patches-applying-proc.adoc:11]: I ASSuME that you are hard-coding 7.9 on purpose for this particular note. If not  - Fix it LOL

**Review by michelehaglund — APPROVED** [review]
One minor comment. Looks good.

File: doc-content/enterprise-only/installation/patches-applying-proc.adoc (hunk 2 above)
Comment (csherrar) [reply doc-content/enterprise-only/installation/patches-applying-proc.adoc:11]: Yes, you assumed right :) This is going into the 7.8 docs and we don't have a future version attribute. (Maybe we should have past and future version attributes? WDYT?)

**Review by karreiro — APPROVED** [review]

**Pull request merged**